nix/shell.nix

```
@@ -0,0 +1,26 @@
+{ pkgs, esy ? pkgs.nodePackages.esy, deku, npmPackages }:
+
+pkgs.mkShell {
+  shellHook = deku.configurePhase;
+
+  nativeBuildInputs = (with pkgs; [
+    # Make developer life easier
+    ## General tooling
+    docker
+    docker-compose
+    nodejs-12_x
+
+    # Nix files formatter
+    nixfmt
+  ]) ++ (with pkgs.ocaml-ng.ocamlPackages_5_00; [
+    # OCaml developer tooling
+    ocaml
+    findlib
+    dune_2
+    # esy
+    ocaml-lsp
```

```suggestion
    ocaml-lsp
    ocamlformat
```
We're missing ocamlformat

---

Hmm, that doesn't work - not defined.

---

Oh, we have to use do something like:
```suggestion
    ocaml-lsp
    pkgs.ocamlformat
```

---

Darn, the version isn't correct.

---

I think I solved this, but I had to set a ocamlformat version